src/CoreLib/deploy.fs

```diff
@@ -159,7 +159,7 @@ type DeploymentSettings() =
     static member val internal MasterPort = 1080 with get, set
     static member val internal ControlPort = 1081 with get, set
     /// Directory at remote node that holds Prajna data files
-    static member val DataFolder = @"OneNetData" with get
+    static member val DataFolder = @"PrajnaData" with get
```

Comment:
For test clusters with existing data, on each machine, I will create a symbolic link PrajnaData that points to original folder